### DISCUSSION TITLE
Advanced search result sort order

### DISCUSSION THREAD

**Discussion by R1BLH** [wavelog/wavelog#711] [Ideas]
Now the advanced search allows you to save an ADIF with the search result sorted by date in descending order.
It would be nice to be able to choose between ascending and descending sorting.
Only the query description can be edited manually, not the query itself, which is created in the constructor, so it is impossible to change the sorting in the saved query.

**Comment by HB9HIL** [comment]
Hei @R1BLH 

We could enable the ability to sort the resultstable in Advanced Search. Or which output do you expect to be sorted in the end?

<img width="184" alt="image" src="https://github.com/user-attachments/assets/364372a0-a0b3-43d1-9c77-9d02bcba431a">

**Reply by R1BLH** [reply]
As a result, I would like to get an ADIF file with the oldest QSOs listed first. Currently, the most recent QSOs are listed first in the ADIF file that is saved when the search results are exported.
This sometimes leads to funny results when applying for awards.
Perhaps the most convenient way would be to add a sort rule (ascending or descending) to the query constructor (similar to the rules for querying by field values).

![изображение](https://github.com/user-attachments/assets/993c149e-9534-4728-921e-ff6b5e696609)
